pallets/liquidity-pools/src/mock.rs

```diff
@@ -26,24 +26,12 @@ pub const CHAIN_ID: u64 = 1;
 pub const ALICE_32: [u8; 32] = [2; 32];
 pub const ALICE: AccountId = AccountId::new(ALICE_32);
 pub const ALICE_ETH: [u8; 20] = [2; 20];
-pub const ALICE_EVM_DOMAIN_ADDRESS: DomainAddress = DomainAddress::EVM(CHAIN_ID, ALICE_ETH);
-// TODO(future): Can be removed after domain conversion refactor
-pub const ALICE_EVM_LOCAL_ACCOUNT: AccountId = {
-	let mut arr = [0u8; 32];
-	let mut i = 0;
-	while i < 20 {
-		arr[i] = ALICE_ETH[i];
-		i += 1;
-	}
-	AccountId::new(arr)
-};
-pub const CENTRIFUGE_DOMAIN_ADDRESS: DomainAddress = DomainAddress::Centrifuge(ALICE_32);
+pub const ALICE_EVM_DOMAIN_ADDRESS: DomainAddress = DomainAddress::Evm(42, ALICE_ETH);
```

Comment:
You re-inserted the incorrect chain ID which was fixed in #1964

```suggestion
pub const ALICE_EVM_DOMAIN_ADDRESS: DomainAddress = DomainAddress::Evm(CHAIN_ID, ALICE_ETH);
```

Reply:
Ok, this has changed because `ALICE_EVM_LOCAL_ACCOUNT` no longer exists, and now we're using `ALICE_EVM_LOCA_ACCOUNT`, which has chain 42, not sure why TBH...

Reply:
Ah ok, never mind... fixing it

Reply:
Thanks for being aware of this!